### ISSUE TITLE
Indent rule reports false: multi-line + fat arrow

### ISSUE THREAD

**Issue by necolas** [eslint/eslint#2239]
Noticed this inconsistency in indentation reporting (v0.18.0).

```js
['a', 'b', 'c', 'd',
    'e', 'f', 'g'].forEach(function (letter) {
  console.log(letter);
});
```

No errors.

```js
['a', 'b', 'c', 'd',
    'e', 'f', 'g'].forEach((letter) => {
  console.log(letter);
});
```

Produces the error:

```
3:3 error  Expected indentation of 6 characters           indent
```

**Comment by nzakas** [comment]
Thanks, will take a look.

**Comment by justin808** [comment]
A related issue if using the airbnb eslint style guidelines:

```
29:17  error    Block must not be padded by blank lines  padded-blocks
```

```javascript
import React from 'react';
import Comment from './Comment';

const CommentList = React.createClass({
  displayName: 'CommentList',

  propTypes: {
    comments: React.PropTypes.array
  },

  render() {
    const reversedData = this.props.comments.slice(0).reverse();
    const commentNodes = reversedData.map((comment, index) => {
      // `key` is a React-specific concept and is not mandatory for the
      // purpose of this tutorial. if you're curious, see more here:
      // http://facebook.github.io/react/docs/multiple-components.html#dynamic-children
      return (
        <Comment author={comment.author} key={index} text={comment.text}/>
      );
    });

    return (
      <div className="commentList">
        {commentNodes}
      </div>
    );
  }
});

export default CommentList;
```

**Comment by nzakas** [comment]
@justin808 that's not related to this issue. Can you open a new issue for that?

**Comment by necolas** [comment]
@nzakas @justin808 i think that's the same as #2211

**Comment by PiPeep** [comment]
I think this is the same issue: https://gist.github.com/PiPeep/c2ace2fd7b82917cb263
